projects/js-packages/ai-client/src/components/message/index.tsx

```diff
@@ -71,8 +81,10 @@ const messageIconsMap = {
 export default function Message( {
 	severity = MESSAGE_SEVERITY_INFO,
 	icon = null,
-	showSidebarIcon = false,
-	onSidebarIconClick = () => {},
+	showAIFeedbackThumbs = false,
+	ratedItem = '',
```

Comment:
GuidelineMessage passes down ratedItem which, in turn, is passed down to AiFeedbackThumbs but, so far, it always matches the `feature` prop (ai-assistant). Could ratedItem hold a value different to `feature`? I'm merely curious since looking at the code it seems like a duped prop value.